gitpod-io/gitpod
# :wrench: Ambiguity of `New Workspace` option & :rocket: new `Settings` option

## Description


:wrench: Ambiguity of `New Workspace ...` option & :rocket: new `Settings` option in Project List Item

## Related Issue(s)

Fixes [Internal Conversastion](https://gitpod.slack.com/archives/C01KPEPNBD0/p1677163519383559?thread_ts=1677151354.576229&cid=C01KPEPNBD0)

## How to test


- Open Preview
- Go to `/projects`
- And Click Menu Item (Three dots) of any Project. See 👀 
    - New `Settings` option
    - Modified `New Workspace ...`

## Release Notes

```release-note
Access Project Settings directly from Projects Page on Project Menu Items
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

> Signed-off-by: Siddhant Khare <siddhant@gitpod.io>
